lambda_functions/transform_lambda.py

```diff
@@ -111,10 +155,17 @@ def get_resource_tags_from_metric(
                 tags = get_tags_from_name(bucket_name, "S3", s3_client)
         elif namespace == "AWS/ES":
             domain_name = dimensions.get("DomainName")
-            client_id = dimensions.get("ClientId")
-            if domain_name.startswith(domain_prefix) and client_id:
-                arn = f"arn:aws-us-gov:es:{region}:{client_id}:domain/{domain_name}"
+            if domain_name.startswith(domain_prefix):
+                arn = f"arn:aws-us-gov:es:{region}:{account_id}:domain/{domain_name}"
                 tags = get_tags_from_arn(arn, es_client)
+        elif namespace == "AWS/RDS":
+            db_name = dimensions.get("DBInstanceIdentifier")
+            # AWS will tell you average for all db of certain classes
+            if db_name is None:
+                return tags
```

Comment:
What is the case we're handling here? The metric has no database name? Are these valid brokered databases?

Reply:
I guess we also get a metric for say all of t3.small instances or m6.large